python/paddle/fluid/backward.py

```diff
@@ -42,6 +42,9 @@
                                 logging.INFO,
                                 fmt='%(asctime)s-%(levelname)s: %(message)s')
 
+_grad_op_id_to_fwd_op = {
```

Comment:
I prefer not using the global variable. Try to pass an argument to the method.

Reply:
Any difference between these two implementations? Is this a best practice? I think global variable is somehow easy to use 😂.

Reply:
Yes, the global variable is somehow easy to implement. The global variable should record the global status but not the local status. The `_grad_op_id_to_fwd_op` should be a map related to a certain program but not all programs.

Reply:
Make sense, just update